benchmarks/extents/extents_perf.cpp

```diff
@@ -0,0 +1,47 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+#include <experimental/mdspan>
+
+#include <benchmark/benchmark.h>
+
+#include "fill.hpp"
+
+using index_type = int;
+
+namespace stdex = std::experimental;
+_MDSPAN_INLINE_VARIABLE constexpr auto dyn = stdex::dynamic_extent;
+
+template <class Extents, class ... Args>
+void extents_construction(benchmark::State& state, Extents, int R, Args ... args) {
+  for (auto _ : state) {
```

Comment:
If you have C++17, you can use `[[maybe_unused]]` here to avoid a build warning.
```suggestion
  for ([[maybe_unused]] auto _ : state) {
```